src/Pages/yeongseoblee/Main/Feeds/CommentItem.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import "./CommentItem.scss";
+
+class CommentItem extends Component {
+  constructor() {
+    super();
+    this.state = {
+      clicked: false,
+    };
+  }
+  clickLike = () => {
+    const { clicked } = this.state;
+    this.setState({
+      clicked: !clicked,
+    });
+  };
+
+  render() {
+    const { clicked } = this.state;
+    return (
+      <li className="commentListItems">
+        <div className="nameAndComment">
+          <div className="commentNames">
+            <a className="userNameATags" href="/">
+              my_name
+            </a>
+          </div>
+          &nbsp;
+          <div className="commentContents">{this.props.valueFromFeed}</div>
+        </div>
+        <span className="commentLikes" onClick={this.clickLike}>
+          <button>
+            <div>
+              <svg
+                aria-label="Like"
+                className={clicked ? "likeBtnRed" : "likeBtnBlack"}
```

Comment:
위에 달았던 코멘트와 유사합니다. 아래와 같은 형태로 고치시고 왜 이렇게 바꾸면 좋을지 생각해보시면 좋겠습니다.
```suggestion
                className={clicked ? "likeBtn red" : "likeBtn black"}
```